lib/rubocop/cop/minitest/focus.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Enforces tests are not focused
+      #
+      # @example
+      #   # bad
+      #   focus test 'foo' do
+      #   end
+      #
+      #   # bad
+      #   focus
+      #   test 'foo' do
+      #   end
+      #
+      #   # good
+      #   test 'foo' do
+      #   end
+      #
+      class Focus < Base
+        extend AutoCorrector
+        include RangeHelp
+
+        MSG = 'Remove `focus` from tests.'
+        RESTRICT_ON_SEND = [:focus].freeze
+
+        def_node_matcher :focused?, <<~PATTERN
+          (send nil? :focus ...)
+        PATTERN
+
+        def on_send(node)
+          return unless focused?(node)
+
+          add_offense(node) do |corrector|
+            if node.child_nodes.empty?
+              corrector.remove(range_by_whole_lines(node.source_range, include_final_newline: true))
+            else
+              corrector.replace(node, node.child_nodes.map(&:source).join(' '))
+            end
+          end
+        end
```

Comment:
Can you update to the following? 

```suggestion
        def on_send(node)
          return unless node.receiver

          range = node.loc.selector

          add_offense(range) do |corrector|
            corrector.remove(range)
          end
        end
```

This change will make the offense highlight range clearer. While the layout may be slightly off after autocorrection, let's leave it to `Layout` cops to handle. So, Please adjust the test code accordingly.